api/datahub.oas3.yml

```diff
@@ -722,6 +860,71 @@ components:
     QueryResponse:
       type: object
 
+    LoginProviders:
+      type: array
+      items:
+        $ref: '#/components/schemas/LoginProvider'
+
+
+    LoginProvider:
+      properties:
+        name:
+          type: string
+          description: Unique identifying name for this provider
+        type:
+          type: string
+          enum:
+            - bearer
+            - basic
+          description: Must be either bearer (jwt) or basic (username+password)
```

Comment:
Should we perhaps be clearer in the naming here. Basic is basic auth towards the external API but jwt is a bit inaccurate. It is an OAuth2 client credential flow. For me bearer or jwt would be if the token was somehow a long lived token stored locally.